### ISSUE TITLE
Tabnine does not work in Jetbrains client when using Code With Me with IdeaVim plugin

### ISSUE THREAD

**Issue by dsychin** [codota/tabnine-intellij#511]
### Issue

When hosting a Code With Me session, the second person connected to the session with Jetbrains client will not see Tabnine suggestions if the host has IdeaVim plugin installed (even if disabled). Tested with Goland and Webstorm.

It works however if the host does not have IdeaVim plugin installed but the client does.

### Steps to reproduce

1. Use the IDE with IdeaVim and Tabnine plugins installed.
2. Start a Code With Me session, copy the link and launch it in the browser to join the session.
3. Type some code in the client, the host will see Tabnine suggestions but the client will not.
4. Uninstall IdeaVim plugin and repeat step 3, both client and host will see Tabnine suggestions.

gz#22984<br><br><i>(related to <a href='https://tabninedesk.zendesk.com/agent/tickets/22984'>Zendesk ticket #22984</a>)</i>

**Comment by TzufTabnine** [comment]
Hi @dsychin 

Can you send us a video of the issue?

**Comment by dsychin** [comment]
Hi @TzufTabnine 
I think the issue is not related to a conflict between IdeaVim and TabNine because it was not working when I tried uninstalling IdeaVim on the host machine again.
I was able to fix the issue by doing a full Goland and Jetbrains client reinstall and making sure the TabNine plugin is also installed on the client.
Previously with the plugin installed on the client, the suggestions were not showing up but I was not able to replicate that issue again.
